pubtools/exodus/_tasks/push.py

```diff
@@ -14,12 +16,12 @@ def add_args(self):
         super(ExodusPushTask, self).add_args()
 
         self.parser.add_argument(
-            "sync",
-            nargs="+",
+            "source",
             help=(
-                """A comma-separated pair of paths to sync: SRC,DEST.\n"""
-                """Where SRC is a local path to a file or directory and DEST\n"""
-                """is the remote destination for sync."""
+                """$PUSHSOURCE_BACKEND:$STAGING_ROOT, where $PUSHSOURCE_BACKEND is a valid """
```

Comment:
This help text isn't right, since by loading the content through pushsource we support *all* pushsource backends and not just those which use a $STAGING_ROOT.  Could it be more general please, like just "Source(s) of content to be pushed" ?  The example could be kept as a hint as to which sources are useful in practice.

Note I'm also maintaining internal doc https://docs.engineering.redhat.com/x/60QiDQ , that's a place which can be used to add more specific detail.  Maybe you want to document the new custom backend there.